### DISCUSSION TITLE
Difference to route discovery package?

### DISCUSSION THREAD

**Discussion by Radiergummi** [spatie/laravel-route-attributes#120] [General]
Hi @freekmurze, I've just found the [laravel-route-discovery](https://github.com/spatie/laravel-route-discovery) package, which pretty much looks like a v2 of this one. What was the motivation behind adding a second one, and do you plan to deprecate this one?

**Comment by freekmurze** [comment]
I don't plan to deprecate either one.

Route attributes package --> discover routes based on attributes your manually add
Route discovery package --> discover routes based on convention, without you having to specify anything.

You could even use both packages at the same time.

Does that make it clear?

**Comment by Radiergummi** [comment]
That definitely helps, although it feels like standalone route attributes are a special case of the discovery package, ie. explicit attributes to customise aspects of a route otherwise configured by convention.  

I'm probably going to use both packages in our application.

Thank you for the reply!
